control

```diff
@@ -1,6 +1,6 @@
 Package: com.ps.youpip
 Name: YouPIP
-Depends: mobilesubstrate, gsc.ipad | com.cabralcole.forceinpicture | com.ps.forceinpicture
+Depends: mobilesubstrate, firmware (>= 14.0) | gsc.ipad | com.cabralcole.forceinpicture | com.ps.forceinpicture
```

Comment:
Why do you need iOS 14 or higher?

Reply:
It's setting a dependency for installing ForceInPicture only if the device is not and iPad and not iOS 14 and higher, since iOS 14 does not require ForceInPicture to work.

On iOS 13 it will still require to install it (tested on another device).

Reply:
> Why do you need iOS 14 or higher?

It's setting a dependency for installing ForceInPicture only if the device is not and iPad and not iOS 14 and higher, since iOS 14 does not require ForceInPicture to work.

On iOS 13 it will still require to install it (tested on another device).